src/test/java/core/basesyntax/service/InputDataServiceImplTest.java

```diff
@@ -0,0 +1,88 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import java.util.List;
+import org.junit.Test;
+
+public class InputDataServiceImplTest {
+    private final InputDataService inputDataService = new InputDataServiceImpl();
+
+    @Test
+    public void stringToFruitTransactionConverter_validInputTextFormat_ok() {
+        String validLineOne = "b,pineapple,10";
+        String validLineTwo = "r,banana,100";
+        String validLineThree = "p,orange,50";
+        List<String> validInputTextList = List.of(validLineOne, validLineTwo, validLineThree);
+        assertEquals(3,
+                inputDataService.stringToFruitTransactionConverter(validInputTextList).size());
+    }
+
+    @Test
+    public void stringToFruitTransactionConverter_invalidDataFormatExceptionMessage_Ok() {
```

Comment:
Lets divide a big test with few assertion to a smaller tests with one assertion, and don't create a lot of test with the same condition. Change it in all test classes.